### ISSUE TITLE
JAWS & IE11: initial "Big Play Button" (BPB) and Play button is silent when tabbing to it

### ISSUE THREAD

**Issue by serenanubi** [videojs/video.js#4583]
## Description
Big play button is silent upon tabbing to it in IE11 with JAWS 18.

## Steps to reproduce
Go to videojs.com and use the tab button until the focus gets to the BPB.

### Expected
Once the focus is on the BPB, JAWS should say "Play Video button".

### Actual
JAWS is silent.

### Error output
N/A

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
Latest version (v6.2.5).
#### browsers
IE11
#### OSes
Windows 7
### plugins
Not sure..

**Comment by gkatsev** [comment]
@OwenEdwards any ideas?

**Comment by waughcbw** [comment]
@gkatsev Any updates?

**Comment by OwenEdwards** [comment]
@gkatsev not sure what's going on here, but I see this behavior on the http://videojs.com/ site with IE11 & JAWS18: if you use arrow keys to move to the BPB, JAWS announces it but says something before "Play Video" (it sounds like "key"); if you Tab to the BPB, JAWS doesn't say anything.

But looking at the HTML, the `<span>` with `aria-hidden="true"` for the icon that you put in is missing. Did it somehow get dropped from the latest version of video.js?

**Comment by serenanubi** [comment]
@gkatsev @OwenEdwards All the a11y improvements are great. It'd be really nice to get this fixed for JAWS & IE11. Any movement on this would be appreciated. I know IE11 has had issues with aria-live, so I was thinking that may have been the issue. 

I only mentioned the example on http://videojs.com, but this is affecting our users as well.

**Comment by OwenEdwards** [comment]
@serenanubi I looked into this a little, and as I noted in #4601, there's some confusion because the http://videojs.com site is actually using v5.x, not v6.x of video.js. So I wanted to confirm that, when you say it's affecting your users, that you're using v6.2.5 as you mentioned in the original issue report?

**Comment by serenanubi** [comment]
Actually we're using v5.x, which has the issue, but I was hoping v6.2.5 would have a fix, so I was testing our codebase with v6.2.5 instead to verify if the issue had been resolved, but it seems to still be a problem. Sorry for the confusion (and assumption on my part).

**Comment by OwenEdwards** [comment]
@serenanubi is this still an issue in v6.x releases? If not, can we close this issue?

**Comment by serenanubi** [comment]
Yes, this is still an issue in v6.x releases.

**Comment by OwenEdwards** [comment]
I'm still seeing this in the latest v6.x build, using JAWS 2018 and IE11. I need to do some more investigation.
